src/compiler/emitter.ts

```diff
@@ -2683,15 +2676,17 @@ if (typeof __param !== "function") __param = function (paramIndex, decorator) {
                 if (!exportEquals && exportSpecifiers && hasProperty(exportSpecifiers, name.text)) {
                     for (let specifier of exportSpecifiers[name.text]) {
                         writeLine();
-                        emitStart(specifier.name);
                         if (compilerOptions.module === ModuleKind.System) {
+                            emitStart(specifier.name);

```

Comment:
can we file a bug for source maps for modules in generarl (AMD, UMD, and System)